ArjunaJTA/jta/classes/com/arjuna/ats/internal/jta/transaction/arjunacore/jca/XATerminatorImple.java

```diff
@@ -95,7 +95,7 @@ public void commit (Xid xid, boolean onePhase) throws XAException
                     .getTransactionImporter().getImportedTransaction(xid);
 
             if (tx == null) {
-                XAException xaException = new XAException(jtaLogger.i18NLogger.get_no_subordinate_txn_for_commit(xid));
+                XAException xaException = new XAException(jtaLogger.i18NLogger.get_no_subordinate_txn_for("commit", xid));
```

Comment:
Shouldn't we deprecate the log message first before removing it?

Reply:
Agreed - let's go back to the original message for this one as people may be using the error code

Reply:
@mmusgrov @tomjenkinson yeap, agree, I returned back the log message.